### DISCUSSION TITLE
Welcome to meme-box Discussions!

### DISCUSSION THREAD

**Discussion by negue** [negue/meme-box#164] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NexShey** [comment]
pourriez-vous  ajouter un system pour que on puisse ajouter du texte sur les vidéo et le modifier

**Reply by negue** [reply]
Hé

(ceci est traduit en utilisant Google)

ce genre de fonctionnalité que je souhaite aussi avoir dans un [avenir proche][future_roadmap], mais actuellement c'est celle qui sera la [prochaine ][next_version].

Une fois la prochaine version publiée, j'essaierai d'abord de développer cette fonctionnalité.


---

English:

Hey

(this is translated using google)

this kind of feature I also want to have in the [near future][future_roadmap], but currently this is the one that will be [next][next_version].

Once the next version is released I'll try to develop this feature first.
 
[next_version]: https://github.com/negue/meme-box?tab=readme-ov-file#roadmap
[future_roadmap]: https://github.com/negue/meme-box/blob/develop/ROADMAP.md
